source/deployment/deployContracts.ts

```diff
@@ -3,8 +3,8 @@
 import * as fs from "async-file";
 import * as getPort from "get-port";
 import * as path from "path";
-import * as EthjsHttpProvider from "ethjs-provider-http";
-import * as EthjsQuery from "ethjs-query";
+import EthjsHttpProvider = require('ethjs-provider-http');
+import EthjsQuery = require("ethjs-query");
```

Comment:
It turns out `import * as EthjsQuery from 'ethjs-query';` is not correct and only works by happenstance and will likely break in the future.  Fixing these up was just some code cleanup.  Note, this style is only necessary for imports that you need to `new` directly.  For an import that is just a property bag, the other style is still appropriate (as far as I understand).

Reply:
That's weird, why does it actually work if it's not correct?

Reply:
Apparently just because of the way TypeScript compiler happens to be implemented makes it so it happens to be valid and they haven't bother to write the code that will trigger an error when someone does this incorrectly.  This Q&A talks a bit about the situation: https://stackoverflow.com/questions/39415661/what-does-resolves-to-a-non-module-entity-and-cannot-be-imported-using-this